### PR TITLE
Production deploy env var setup

### PR DESCRIPTION
**Pull request by johnmosesman**
Haven't run this yet so it may need to be tweaked a little. Looks for a `.env.prod` when running prod scripts

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ import "solidity-coverage";
 import "hardhat-deploy";
 import "@nomiclabs/hardhat-ethers";
 
-dotenv.config();
+const envFile = process.env.DEPLOY_ENV === "production" ? ".env.prod" : ".env";
```
Comment (vanb) [inline hardhat.config.ts:12]: Can we use existing `APP_ENV` here?

Comment (vanb) [reply hardhat.config.ts:12]: Or I guess this is just for build time, maybe best to keep separate

Comment (johnmosesman) [reply hardhat.config.ts:12]: Or are you saying set `APP_ENV=prod npx hardhat ...` in the script?

Comment (johnmosesman) [reply hardhat.config.ts:12]: (from discord: leaving as is)

**Pull request merged**